README.md

```diff
@@ -93,14 +93,22 @@ Feel free to submit a PR if you want to add items to the Roadmap!
 - [ ] Integrations with brokers
 
 
-
 # How To Use
 
 ```bash
 git clone https://github.com/GoPlan-Finance/GoPlan-app.git
 cd goplan-app
 ```
 
+### Prerequisites
+
+- Node 8 or newer
```

Comment:
Not sure about `v8`, the older version I tested was `v14`

`v8` dates back I. 2017 I think

Reply:
I just used the specs from parse, but your probably right, we should put 14 here since we havent tested anything else. I have had some issues with v17 and reverted back to 16.13.1. Which version are you runnning?